Lib/test/test_timeit.py

```diff
@@ -4,7 +4,7 @@
 import io
 from textwrap import dedent
 
-from test.support import captured_stdout
+from test.support import captured_stdout, force_not_colorized
 from test.support import captured_stderr
```

Comment:
You can merge those 3 imports into one:


```suggestion
from test.support import (
    captured_stderr, captured_stdout, force_not_colorized,
)
```

Sorry I missed reviewing this part.